### ISSUE TITLE
Panel from cf is slow to load

### ISSUE THREAD

**Issue by jsignell** [pyviz/panel#407]
With a fresh environment created using `conda create --name panel-cf -c conda-forge panel notebook`, when I open a notebook and run:

```python
%%time

import panel as pn
pn.extension()

display(pn.widgets.Button(name='Click me'))
```

I get:
```
CPU times: user 540 ms, sys: 188 ms, total: 728 ms
Wall time: 16.9 s
```

<details>

```
# packages in environment at /Users/jsignell/conda/envs/panel-cf:
#
# Name                    Version                   Build  Channel
appnope                   0.1.0                 py37_1000    conda-forge
asn1crypto                0.24.0                py37_1003    conda-forge
attrs                     19.1.0                     py_0    conda-forge
backcall                  0.1.0                      py_0    conda-forge
bleach                    3.1.0                      py_0    conda-forge
bokeh                     1.1.0                    py37_0    conda-forge
bzip2                     1.0.6             h1de35cc_1002    conda-forge
ca-certificates           2019.3.9             hecc5488_0    conda-forge
certifi                   2019.3.9                 py37_0    conda-forge
cffi                      1.12.3           py37hccf1714_0    conda-forge
chardet                   3.0.4                 py37_1003    conda-forge
cryptography              2.6.1            py37h212c5bf_0    conda-forge
decorator                 4.4.0                      py_0    conda-forge
defusedxml                0.5.0                      py_1    conda-forge
entrypoints               0.3                   py37_1000    conda-forge
freetype                  2.10.0               h24853df_0    conda-forge
idna                      2.8                   py37_1000    conda-forge
ipykernel                 5.1.0           py37h24bf2e0_1002    conda-forge
ipython                   7.5.0            py37h24bf2e0_0    conda-forge
ipython_genutils          0.2.0                      py_1    conda-forge
jedi                      0.13.3                   py37_0    conda-forge
jinja2                    2.10.1                     py_0    conda-forge
jpeg                      9c                h1de35cc_1001    conda-forge
jsonschema                3.0.1                    py37_0    conda-forge
jupyter_client            5.2.4                      py_3    conda-forge
jupyter_core              4.4.0                      py_0    conda-forge
libblas                   3.8.0                8_openblas    conda-forge
libcblas                  3.8.0                8_openblas    conda-forge
libcxx                    8.0.0                         2    conda-forge
libcxxabi                 8.0.0                         2    conda-forge
libffi                    3.2.1             h6de7cb9_1006    conda-forge
libgfortran               3.0.1                         0    conda-forge
liblapack                 3.8.0                8_openblas    conda-forge
libpng                    1.6.37               h2573ce8_0    conda-forge
libsodium                 1.0.16            h1de35cc_1001    conda-forge
libtiff                   4.0.10            h79f4b77_1001    conda-forge
markdown                  2.6.11                     py_0    conda-forge
markupsafe                1.1.1            py37h1de35cc_0    conda-forge
mistune                   0.8.4           py37h1de35cc_1000    conda-forge
nbconvert                 5.5.0                      py_0    conda-forge
nbformat                  4.4.0                      py_1    conda-forge
ncurses                   6.1               h0a44026_1002    conda-forge
notebook                  5.7.8                    py37_0    conda-forge
numpy                     1.16.3           py37hdf140aa_0    conda-forge
olefile                   0.46                       py_0    conda-forge
openblas                  0.3.6                hd44dcd8_1    conda-forge
openssl                   1.1.1b               h01d97ff_2    conda-forge
packaging                 19.0                       py_0    conda-forge
pandoc                    2.7.2                         0    conda-forge
pandocfilters             1.4.2                      py_1    conda-forge
panel                     0.5.1                         1    conda-forge
param                     1.9.0                      py_0    conda-forge
parso                     0.4.0                      py_0    conda-forge
pexpect                   4.7.0                    py37_0    conda-forge
pickleshare               0.7.5                 py37_1000    conda-forge
pillow                    6.0.0            py37h7095ceb_0    conda-forge
pip                       19.1                     py37_0    conda-forge
prometheus_client         0.6.0                      py_0    conda-forge
prompt_toolkit            2.0.9                      py_0    conda-forge
ptyprocess                0.6.0                   py_1001    conda-forge
pycparser                 2.19                     py37_1    conda-forge
pyct                      0.4.6                      py_0    conda-forge
pyct-core                 0.4.6                      py_0    conda-forge
pygments                  2.3.1                      py_0    conda-forge
pyopenssl                 19.0.0                   py37_0    conda-forge
pyparsing                 2.4.0                      py_0    conda-forge
pyrsistent                0.15.1           py37h01d97ff_0    conda-forge
pysocks                   1.6.8                 py37_1002    conda-forge
python                    3.7.3                h0d93f26_0    conda-forge
python-dateutil           2.8.0                      py_0    conda-forge
pyviz_comms               0.7.2                      py_0    conda-forge
pyyaml                    5.1              py37h1de35cc_0    conda-forge
pyzmq                     18.0.1           py37h2d07e9b_1    conda-forge
readline                  7.0               hcfe32e1_1001    conda-forge
requests                  2.21.0                py37_1000    conda-forge
send2trash                1.5.0                      py_0    conda-forge
setuptools                41.0.1                   py37_0    conda-forge
six                       1.12.0                py37_1000    conda-forge
sqlite                    3.26.0            h1765d9f_1001    conda-forge
terminado                 0.8.2                    py37_0    conda-forge
testpath                  0.3.1                    py37_1    conda-forge
tk                        8.6.9             ha441bb4_1001    conda-forge
tornado                   6.0.2            py37h01d97ff_0    conda-forge
traitlets                 4.3.2                 py37_1000    conda-forge
urllib3                   1.24.2                   py37_0    conda-forge
wcwidth                   0.1.7                      py_1    conda-forge
webencodings              0.5.1                      py_1    conda-forge
wheel                     0.33.1                   py37_0    conda-forge
xz                        5.2.4             h1de35cc_1001    conda-forge
yaml                      0.1.7             h1de35cc_1001    conda-forge
zeromq                    4.3.1             h0a44026_1000    conda-forge
zlib                      1.2.11            h1de35cc_1004    conda-forge
```
</details>

In an environment depending on panel from pyviz (`create --name panel-pyviz -c pyviz panel notebook`), I get:
```
CPU times: user 448 ms, sys: 162 ms, total: 610 ms
Wall time: 840 ms
```

<details>

```
# packages in environment at /Users/jsignell/conda/envs/panel-pyviz:
#
# Name                    Version                   Build  Channel
appnope                   0.1.0                    py37_0
asn1crypto                0.24.0                   py37_0
attrs                     19.1.0                   py37_1
backcall                  0.1.0                    py37_0
blas                      1.0                         mkl
bleach                    3.1.0                    py37_0
bokeh                     1.1.0                    py37_0
ca-certificates           2019.1.23                     0
certifi                   2019.3.9                 py37_0
cffi                      1.12.3           py37hb5b8e2f_0
chardet                   3.0.4                    py37_1
cryptography              2.6.1            py37ha12b0ac_0
decorator                 4.4.0                    py37_1
defusedxml                0.6.0                      py_0
entrypoints               0.3                      py37_0
freetype                  2.9.1                hb4e5f40_0
idna                      2.8                      py37_0
intel-openmp              2019.3                      199
ipykernel                 5.1.0            py37h39e3cac_0
ipython                   7.5.0            py37h39e3cac_0
ipython_genutils          0.2.0                    py37_0
jedi                      0.13.3                   py37_0
jinja2                    2.10.1                   py37_0
jpeg                      9b                   he5867d9_2
jsonschema                3.0.1                    py37_0
jupyter_client            5.2.4                    py37_0
jupyter_core              4.4.0                    py37_0
libcxx                    4.0.1                hcfea43d_1
libcxxabi                 4.0.1                hcfea43d_1
libedit                   3.1.20181209         hb402a30_0
libffi                    3.2.1                h475c297_4
libgfortran               3.0.1                h93005f0_2
libpng                    1.6.37               ha441bb4_0
libsodium                 1.0.16               h3efe00b_0
libtiff                   4.0.10               hcb84e12_2
markdown                  3.1                      py37_0
markupsafe                1.1.1            py37h1de35cc_0
mistune                   0.8.4            py37h1de35cc_0
mkl                       2019.3                      199
mkl_fft                   1.0.12           py37h5e564d8_0
mkl_random                1.0.2            py37h27c97d8_0
nbconvert                 5.5.0                      py_0
nbformat                  4.4.0                    py37_0
ncurses                   6.1                  h0a44026_1
notebook                  5.7.8                    py37_0
numpy                     1.16.3           py37hacdab7b_0
numpy-base                1.16.3           py37h6575580_0
olefile                   0.46                     py37_0
openssl                   1.1.1b               h1de35cc_1
packaging                 19.0                     py37_0
pandoc                    2.2.3.2                       0
pandocfilters             1.4.2                    py37_1
panel                     0.5.1                      py_0    pyviz
param                     1.9.0                      py_0    pyviz
parso                     0.4.0                      py_0
pexpect                   4.7.0                    py37_0
pickleshare               0.7.5                    py37_0
pillow                    6.0.0            py37hb68e598_0
pip                       19.1                     py37_0
prometheus_client         0.6.0                    py37_0
prompt_toolkit            2.0.9                    py37_0
ptyprocess                0.6.0                    py37_0
pycparser                 2.19                     py37_0
pyct                      0.4.6                      py_0    pyviz
pyct-core                 0.4.6                      py_0    pyviz
pygments                  2.3.1                    py37_0
pyopenssl                 19.0.0                   py37_0
pyparsing                 2.4.0                      py_0
pyrsistent                0.14.11          py37h1de35cc_0
pysocks                   1.6.8                    py37_0
python                    3.7.3                h359304d_0
python-dateutil           2.8.0                    py37_0
pyviz_comms               0.7.2                      py_0    pyviz
pyyaml                    5.1              py37h1de35cc_0
pyzmq                     18.0.0           py37h0a44026_0
readline                  7.0                  h1de35cc_5
requests                  2.21.0                   py37_0
send2trash                1.5.0                    py37_0
setuptools                41.0.1                   py37_0
six                       1.12.0                   py37_0
sqlite                    3.28.0               ha441bb4_0
terminado                 0.8.2                    py37_0
testpath                  0.3.1                    py37_0
tk                        8.6.8                ha441bb4_0
tornado                   6.0.2            py37h1de35cc_0
traitlets                 4.3.2                    py37_0
urllib3                   1.24.2                   py37_0
wcwidth                   0.1.7                    py37_0
webencodings              0.5.1                    py37_1
wheel                     0.33.1                   py37_0
xz                        5.2.4                h1de35cc_4
yaml                      0.1.7                hc338f04_2
zeromq                    4.3.1                h0a44026_3
zlib                      1.2.11               h1de35cc_3
zstd                      1.3.7                h5bba6e5_0
```
</details>

**Comment by jsignell** [comment]
This came up in the context of the intake GUI which runs `pn.extension()` on `import intake`

**Comment by philippjfr** [comment]
Looks like it's not satisfying all the build dependencies in the latest conda-forge builds. See [the logs](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=32009):

> Custom model compilation failed with: No module named numpy

We should fix this asap.

**Comment by philippjfr** [comment]
Fixed in https://github.com/conda-forge/panel-feedstock/pull/8, new build will hopefully be available soon.

**Comment by henriqueribeiro** [comment]
Sorry to re-open this, but I'm having a similar problem. Running the code above on a restarted kernel I get the following times:
```
CPU times: user 1.03 s, sys: 871 ms, total: 1.9 s
Wall time: 25 s
```

Do you have any guess of what may be causing this?
Thanks

**Comment by philippjfr** [comment]
You'd have to tell me where you installed panel from.

**Comment by henriqueribeiro** [comment]
A regular pip install.

```
$ pip show panel
Name: panel
Version: 0.5.1
Summary: A high level dashboarding library for python visualization libraries.
Home-page: http://pyviz.org
Author: PyViz developers
Author-email: developers@pyviz.org
License: BSD
Location: /usr/local/lib/python3.7/site-packages
Requires: pyviz-comms, testpath, pyct, markdown, bokeh, param
Required-by:
```

**Comment by philippjfr** [comment]
Thanks, that seems like a real issue, the cached model files are indeed not packaged up.

**Comment by henriqueribeiro** [comment]
@philippjfr Do you want me to test something?

**Comment by henriqueribeiro** [comment]
Should I re-open a new issue?

**Comment by philippjfr** [comment]
Please do, this will be a blocker for a new release.

**Comment by henriqueribeiro** [comment]
Thanks @philippjfr
